cypress/integration/views/reports.view.ts

```diff
@@ -23,3 +23,4 @@ export const closeRowIdentifiedRisk =
 export const switchToggle = ".pf-c-switch__toggle";
```

Comment:
This file also needs to be renamed to something like generalConfig.view.ts just like you renamed the model file in this PR.

Reply:
Thanks @nachandr I've added created separate view files for `general cypress/integration/views/general.view.ts`